nixos/modules/services/home-automation/zwave-js.nix

```diff
@@ -0,0 +1,150 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+  cfg = config.services.zwave-js;
+  cacheDir = "/var/cache/zwave-js";
+  runtimeDir = "/run/zwave-js";
+  defaultConfig = {
+    storage = {
+      inherit cacheDir;
+    };
+  };
+  mergedConfigFile = "${runtimeDir}/config.json";
```

Comment:
Also looks ike runtimeDir is only used here, so spending a variable on it doesn't seem very useful.